index.js

```diff
@@ -27,6 +28,22 @@ const compile = function(options) {
   console.log(`Closure Compiler version: ${gccPackage.version}`);
 
   const compiler = new Compiler(options);
+
+  // use native versions so we don't have a dep on Java
+  compiler.JAR_PATH = null;
+  compiler.javaPath = getNativeImagePath();
+
+  if (!compiler.javaPath) {
```

Comment:
Would it be reasonable to fall back on the Java version if the native binary can't be found?

Reply:
Yup, could do that.